openedx/features/course_experience/templates/course_experience/course-updates-fragment.html

```diff
@@ -36,7 +36,22 @@
         </div>
     </header>
     <div class="page-content">
-        ${HTML(updates_html)}
+        % if len(update_list_html) > 0:
+            ${HTML(update_list_html)}
+        % else:
+            <div class="all-updates">
+                % for index, update in enumerate(updates):
+                <article class="updates-article">
+                    % if not update.get("is_error"):
+                    <div class="date">${update.get("date")}</div>
+                    % endif
+                    <div class="article-content">
+                        ${HTML(update.get("content"))}
+                    </div>
+                </article>
```

Comment:
@dianakhuang the problem with the old logic is that it didn't include the dates. I mistakenly thought that the rendered HTML was exactly what was used on the old updates page, but @HarryRein discovered that there were two code paths where only one included dates.